hlx_statics/scripts/lib-adobeio.js

```diff
@@ -189,6 +188,19 @@ export function decorateInlineCodes(element) {
   });
 }
 
+/**
+ * Decorates all nested codes in a container element.
+ * @param {Element} element container element
+ */
+export function decorateNestedCodes(element) {
```

Comment:
attempt to re-add the fix in https://github.com/AdobeDocs/adp-devsite/pull/22/commits/9f6bd4dff4a1152580b728ad56f650e783016ac1#r1878692073